internal/csi-addons/rbd/reclaimspace.go

```diff
@@ -90,12 +99,14 @@ func (rscs *ReclaimSpaceControllerServer) ControllerReclaimSpace(
 // of CSI-addons reclaimspace controller service spec.
 type ReclaimSpaceNodeServer struct {
 	*rs.UnimplementedReclaimSpaceNodeServer
+	// Embed NodeServer as it implements helper functions
+	*rbdutil.NodeServer
```

Comment:
same here, it would be cleaner to pass `*util.VolumeLocks` and have that as an attribute of this struct.